app/views/annotation_categories/index.html.erb

```diff
@@ -40,79 +40,74 @@
   </div>
 </div>
 
+<%# Upload modal %>
```

Comment:
I think we talked about this before, but you should work on moving this whole piece into a new partial.